python/custreamz/custreamz/deployment/custreamz_on_kubernetes/Readme.md

```diff
@@ -0,0 +1,48 @@
+**cuStreamz Kubernetes Setup**
+
+*"The following instruction set is for setting up the cuStreamz pipeline on Google Kubernetes Engine (GKE). But, the overall process for deploying cuStreamz 
+on any Kubernetes cluster would be the same."*
+ — cuStreamz team
+
+
+**Step 1: Creating a Kubernetes Cluster**
+1. Create a single-node Kubernetes cluster using the GPU-accelerated Computing option on the Google Kubernetes Engine console page. 
+2. We are using 32 vCPU cores and 2 NVIDIA T4 GPUs for the node. 
```

Comment:
You're right, but I just wanted to conform to the number of Dask workers in the worker-spec.yaml (8 bi-threaded workers), and the scaling-up the worker pod to 2 (to exhaust all 32 cores). This is so that the people going through the instruction set might understand what they can tweak.